test/specs/apexReplayDebugger.e2e.ts

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2023, salesforce.com, inc.
+ * Copyright (c) 2024, salesforce.com, inc.
```

Comment:
This test was created last year, so 2023 was correct.

Reply:
Oh yeah, I was testing the header feature against the file so updated this. I think it does not hurt though. Thanks for catching this.